tests/ui/lto/auxiliary/dylib.rs

```diff
@@ -1,4 +1,4 @@
-//@ compile-flags: -Z thinlto -C codegen-units=8
+//@ compile-flags: -Clto=thin -C codegen-units=8
```

Comment:
To preserve behavior, shouldn't these drop the flag entirely? Otherwise they will switch from thin-local to thin?